crates/aiken-lang/src/gen_uplc.rs

```diff
@@ -1498,8 +1506,8 @@ impl<'a> CodeGenerator<'a> {
                 vec![fst_name.clone(), snd_name.clone()],
                 tipo.clone(),
                 AirTree::local_var(pair_name, tipo.clone()),
-                None,
-                false,
+                msg_func.clone(),
+                msg_func.is_some(),
```

Comment:
These two seem redundant? If we're already passing in the `msg_func`, can't the caller simply rely on that?

Reply:
In this particular case yes, but you will notice for the same function on other cases no